src/main/kotlin/io/github/inductiveautomation/kindling/log/LogNameList.kt

```diff
@@ -0,0 +1,120 @@
+package io.github.inductiveautomation.kindling.log
+
+import com.formdev.flatlaf.extras.FlatSVGIcon
+import io.github.inductiveautomation.kindling.utils.Action
+import io.github.inductiveautomation.kindling.utils.FilterComparator
+import io.github.inductiveautomation.kindling.utils.FilterList
+import io.github.inductiveautomation.kindling.utils.FilterModel
+import io.github.inductiveautomation.kindling.utils.FlatScrollPane
+import io.github.inductiveautomation.kindling.utils.listCellRenderer
+import net.miginfocom.swing.MigLayout
+import javax.swing.ButtonGroup
+import javax.swing.JPanel
+import javax.swing.JToggleButton
+
+class LogNameList(private val emptyLabel: String) : FilterList(emptyLabel) {
+
+    var isShowFullLoggerNames = false
+        set(value) {
+            field = value
+            updateCellRenderer()
+        }
+
+    init {
+        updateCellRenderer()
+    }
+
+    private fun updateCellRenderer() {
+        cellRenderer = listCellRenderer<Any?> { _, value, _, _, _ ->
+            text = when (value) {
```

Comment:
No more tooltips on these entries?

Reply:
Oops, it looks like they got removed in the refactor. I can throw them back in.